MidwestAccessCoalition/lockbox_rails
# basic error page styling for 404s and 500s

## Changelog
- removed static error pages
- routed 404s and 500s to controller actions
- made erb pages for 404s and 500s

## Link to issue:  
Fixes issue #267


## Steps for QA/Special Notes:
- You might need to set `config.consider_all_requests_local = false` to see these locally

## Relevant Screenshots: 
- super important for UI tasks!


## Are you ready for review?:
